js/react-native/README.md

```diff
@@ -0,0 +1,486 @@
+```
+
+### Manage user sessions
+
+The SDK supports two types of user sessions: **Manual Session** for authenticated users and **Anonymous Session** for temporary users.
+
+#### Session types
+
+**1. Manual Session (ManualSessionInfo):**
+Use this when you have an authenticated user with a specific user ID and session token.
+
+```tsx
+import { ManualSessionInfo } from '@sendbird/ai-agent-messenger-react-native';
+
+<AIAgentProviderContainer
+  appId={'YOUR_APP_ID'}
+  aiAgentId={'YOUR_AI_AGENT_ID'}
+  nativeModules={nativeModules}
+  userSessionInfo={new ManualSessionInfo({
+    userId: 'user_id',
+    authToken: 'auth_token',
+    sessionHandler: {
+      onSessionTokenRequired: async (resolve, reject) => {
+        try {
+          const response = await fetch('new-token-endpoint');
+          resolve(response.token);
+        } catch (error) {
+          reject(error);
+        }
+      },
+      onSessionClosed: () => { },
+      onSessionError: (error) => { },
+      onSessionRefreshed: () => { }
+    }
+  })}
+>
+  <FixedMessenger />
+</AIAgentProviderContainer>
+```
+
+**2. Anonymous Session (AnonymousSessionInfo):**
+Use this when you don't have user authentication or want to allow guest access. The server will automatically create a temporary user.
+
+```tsx
+import { AnonymousSessionInfo } from '@sendbird/ai-agent-messenger-react-native';
+
+<AIAgentProviderContainer
+  appId={'YOUR_APP_ID'}
+  aiAgentId={'YOUR_AI_AGENT_ID'}
+  nativeModules={nativeModules}
+  userSessionInfo={new AnonymousSessionInfo()}
+>
+  <FixedMessenger />
+</AIAgentProviderContainer>
+```
+
+#### Authentication
+
+You can manually control user authentication using `useMessengerSessionContext`. Call `authenticate()` before showing the messenger to avoid loading delays.
+
+```tsx
+import { useMessengerSessionContext } from '@sendbird/ai-agent-messenger-react-native';
+
+const App = () => {
+  const { authenticate, sdkUser } = useMessengerSessionContext();
+
+  useEffect(() => {
+    authenticate();
+  }, []);
+
+  return <FixedMessenger />;
+};
+```
+
+#### Deauthentication
+
+To log out the current user, call `deauthenticate()`. This disconnects the user session and clears the authentication state.
+
+```tsx
+import { useMessengerSessionContext } from '@sendbird/ai-agent-messenger-react-native';
+
+const LogoutButton = () => {
+  const { deauthenticate } = useMessengerSessionContext();
+
+  const handleLogout = () => {
+    deauthenticate();
+  };
+
+  return <Button title={'Logout'} onPress={handleLogout} />;
+};
+```
+
+---
+
+## Advanced Features
+
+The following are available advanced features.
+
+### Customizing Theme
+
+You can customize the messenger theme by passing a `theme` prop to `AIAgentProviderContainer`.
+
+```tsx
+<AIAgentProviderContainer
+  appId={'YOUR_APP_ID'}
+  aiAgentId={'YOUR_AI_AGENT_ID'}
+  nativeModules={nativeModules}
+  userSessionInfo={new AnonymousSessionInfo()}
+  theme={{
+    mode: 'dark', // 'light' or 'dark'
+    palette,
```

Comment:
As we don't have `palette` guide here, should be omit this for now and add a separate section later?